cmd/utils/flags.go

```diff
@@ -62,6 +62,21 @@ var (
 			EnvVars: []string{"DB_DBNAME"},
 		},
 	}
+	// s3 flags
+	S3Flags = []cli.Flag{
+		&cli.StringFlag{
+			Name:    "s3.bucket",
```

Comment:
Thank you very much Brave!
Let's make Name and EnvVars to same! I think we can use package name.

services.s3.bucket
SERVICE**S**_S3_BUCKET

I'm sorry to make you work twice.

Reply:
Oh, I am sorry. Actually, I missed code convention here.
I have updated!